packages/core/README.md

```
@@ -0,0 +1,14 @@
+# @realm-walker/core ⚙️
+
+The "Body" of the realm. A headless, deterministic Entity Component System (ECS) powered by Miniplex.
+
+## Key Components
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)